java/client/src/main/java/glide/ffi/resolvers/RedisValueResolver.java

```diff
@@ -4,10 +4,12 @@
 import response.ResponseOuterClass.Response;
 
 public class RedisValueResolver {
+    public static final long MAX_REQUEST_ARGS_LENGTH;
```

Comment:
"Length" is ambiguous in the context (could be number of arguments, number of characters, or memory size).  Consider adding typing to make it more explicit, e.g.
```suggestion
    public static final long MAX_REQUEST_ARGS_LENGTH_IN_BYTES;
```

Reply:
Yeah I agree. I'll update this and the JNI function name. Also I'll clarify why we use JNI instead of just hard-coding a constant.